### DISCUSSION TITLE
How to count BGCs?

### DISCUSSION THREAD

**Discussion by mars188** [antismash/antismash#411] [Q&A]
Hello, 

I ran antiSMASH and it worked awesome. Now I have all those output files (.gbk, json, txt) and folders (region1, clusterblast, knowncluster, smcogs) from each sample. 

How can I count the BGCs (biosynthetic gene clusters) from each sample? 

Many thanks,

**Comment by SJShaw** [comment]
If by "BGC" you mean antiSMASH "region", then you can do something as simple as `find . -name '*.region???.gbk' | wc -l` in the directory containing all the output directories.

If you mean protocluster, then it's slightly more complex, but still fairly simple (this uses BASH, you may need to convert it if your shell is different): `for gbk in */*.region???.gbk; do grep ' protocluster ' $gbk; done | wc -l`
